src/controllers/types.ts

```
@@ -25,8 +25,16 @@ import type {
   Attachment,
   KeyType,
   JsonLdCredentialFormat,
+  JsonObject,
+  W3cJsonLdVerifyCredentialOptions,
+  DataIntegrityProofOptions,
+  W3cJsonLdSignCredentialOptions,
+  W3cCredential,
+  W3cCredentialSubject,
 } from '@credo-ts/core'
+import type { SingleOrArray } from '@credo-ts/core/build/utils'
 import type { DIDDocument } from 'did-resolver'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)